picard/coverart/providers/caa.py

```diff
@@ -123,8 +72,97 @@ def caa_url_fallback_list(desired_size, thumbnails):
     return urls
 
 
+class arrow_button(QtWidgets.QPushButton):
+    """Standard arrow button for CAA image type selection dialog.
+
+    Keyword Arguments:
+        label {string} -- Label to display on the button
+        command {command} -- Command to execute when the button is clicked (default: {None})
+        parent {[type]} -- Parent of the QPushButton object being created (default: {None})
+    """
+
+    ARROW_BUTTON_WIDTH = 35
+    ARROW_BUTTON_HEIGHT = 20
+
+    def __init__(self, label, command=None, parent=None ):
```

Comment:
There's an extra space before last parenthesis ;)

Reply:
And that's something I even look for...  Argh!  :blush: